.gitignore

```diff
@@ -40,4 +40,3 @@ content/doc/developer/extensions/*.adoc
 vendor/
 /.config/
 /.npm/
-/package-lock.json
```

Comment:
I explicitly didn't check this in because I didn't want to bother keeping our javascript packages up to date (latest all the time, whee).


Are you volunteering to keep these packages reasonably up to date? 😺

Reply:
@rtyler 
For the trade off of stability, yes.  
Also, define "reasonable".

Reply:
I'd add the Gemfile.lock to this.